doc/site/zh-cn/guides/thread-primer/network-discovery.md

```diff
@@ -0,0 +1,254 @@
+# 网络发现与形成
+
+## Thread 网络
+
+Thread 网络由三个唯一的标识符标识：
+
+* 2 字节的 PAN ID（Personal Area Network ID，个域网标识符）
+* 8 字节的 XPAN ID（Extended Personal Area Network ID，扩展个域网标识符）
+* 方便人类阅读的 Network Name（网络名称）
+
+例如，一个 Thread 网络可能具有以下标识符：
+
+| 标识符       | 值                   |
+| ------------ | -------------------- |
+| PAN ID       | `0xBEEF`             |
+| XPAN ID      | `0xBEEF1111CAFE2222` |
+| Network Name | `yourThreadCafe`     |
+
+<figure class="attempt-right">
+<a href="../images/ot-primer-network-active-scan_2x.png"><img src="../images/ot-primer-network-active-scan.png" srcset="../images/ot-primer-network-active-scan.png 1x, ../images/ot-primer-network-active-scan_2x.png 2x" border="0" alt="OT Active Scan" /></a>
+</figure>
+
+在创建新的 Thread 网络或搜索现有的网络时，Thread 设备将对无线电范围内的 802.15.4 网络执行主动扫描：
+
+1. 设备在特定 Channel 上广播 802.15.4 信标请求（Beacon Request）。
+2. 范围内的所有 Router 或 REED 都会广播包含其 Thread 网络 PAN ID、XPAN ID 和 Network Name 的信标，以作为回应。
+3. 设备为每个 Channel 重复前两个步骤。
+
+Thread 设备发现范围内的所有网络后，可以选择连接到现有的网络，也可以在未发现任何网络的情况下创建新的网络。
+
+<h2 style="clear:right">Mesh Link Establishment</h2>
+
+Thread 使用 MLE（Mesh Link Establishment）协议来配置链接并将网络的相关信息传播到 Thread 设备。
+
+在链接配置中，MLE 用于：
+
+* 发现相邻设备的链接
+* 确认到相邻设备的链接质量
+* 建立到相邻设备的链接
+* 与对端协商链接参数（设备类型、帧计数器、超时）
+
+MLE 将以下类型的信息传播给希望建立链接的设备：
+
+* Leader data（Leader RLOC, Partition ID（分区标识符）, Partition weight（分区权重））
+* Network data（on-mesh prefixes, address autoconfiguration（地址自动配置）, more-specific routes（具体路由））
+* Route propagation（路由传播）
+
+Thread 中路由传播的工作原理类似于 RIP（Routing Information Protocol，路由信息协议），RIP 是一种距离矢量路由协议。
+
+Note: 仅当 Thread 设备通过 Thread Commissioning 获得 Thread 网络凭据后，MLE 才会继续进行。Commissioning 和安全性将在本入门教程的后文中深入介绍。目前，假定设备已 commissioned。
+
+## 创建新网络
+
+如果设备选择创建新网络，它将选择最不繁忙的 Channel 和其他网络未使用的 PAN ID，然后成为 Router 并选举自己为 Leader。该设备将 MLE Advertisement 消息发送到其他 802.15.4 设备，以通知其链接状态，并响应其他执行主动扫描的 Thread 设备所发出的信标请求。
+
+## 加入现有网络
+
+如果设备选择加入到现有的网络，则会通过 Thread Commissioning 将其 Channel、PAN ID、XPAN ID 和 Network Name 配置为与目标网络相同，然后进行 MLE Attach 过程以作为子节点（End Device）进行加入。此过程用于“父子链接（Child-Parent link）”。
```

Comment:
Here it seems to me that the critical `masterkey` is missed in the original english source. @jwhui @Vyrastas 

>If the device elects to join an existing network, it configures its Channel, PAN
ID, XPAN ID, and Network Name to match that of the target network via Thread
Commissioning

Reply:
I have a question. `master key` == `network key`?

Reply:
yes, Network Master Key sometime be referred as master key or network key.

Reply:
Why not use `network key` in a unified manner, which may make people mistakenly believe that they are two kinds of keys.